lib/grpc/connection.js

```diff
@@ -163,22 +169,27 @@ class GrpcConnection extends EventEmitter {
       }
     }
 
-    // check environment variables for testing parameters and
-    // pass to server via meta-data.
-    const flaky = parseInt(env.NEWRELIC_GRPCCONNECTION_METADATA_FLAKY, 10)
-    const delay = parseInt(env.NEWRELIC_GRPCCONNECTION_METADATA_DELAY, 10)
-    if (flaky) {
-      logger.trace('Adding flaky metadata: %s', flaky)
-      metadata.add('flaky', flaky)
-    }
-    if (delay) {
-      logger.trace('Adding delay metadata: %s', delay)
-      metadata.add('delay',delay)
-    }
+    this._getTestMetadata(metadata, env)
 
     return metadata
   }
 
+  /**
+   * Adds test metadata used to simulate connectivity issues
+   * when appropriate env vars are set
+   *
+   * @param {Metadata} metadata
+   */
+  _getTestMetadata(metadata, env) {
```

Comment:
minor but would prob rename to `_setTestMetadata` given this is mutating the `metadata` collection VS just returning new items.

Reply:
good call